Projects/VS2017/OpenFodder.vcxproj

```diff
@@ -69,22 +69,22 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <LinkIncremental>true</LinkIncremental>
-    <IncludePath>{SDL2 location}\include;{SDL2_mixer location}\include;..\..\Source\Utils;$(IncludePath)</IncludePath>
+    <IncludePath>{SDL2 location}\include;{SDL2_mixer location}\include;{rapidjson location}\include;$(IncludePath)</IncludePath>
```

Comment:
Did you mean to add the rapidjson to VS2017?

Reply:
Yes it's actually unnecessary, VS2015 project too.